src/lib/viewers/doc/_docBase.scss

```diff
@@ -13,6 +13,24 @@ $thumbnail-sidebar-width: 226px;
 }
 
 .bp {
+    &.bp-annotations-discoverable {
+        .textLayer {
```

Comment:
Will need a separate ticket to address removing similar styling code from annotations repo.

Reply:
Done